application/static/landregistry/assets/stylesheets/application.scss

```diff
@@ -22,4 +22,19 @@ float: left;
 .verfied-bullet-numbers-fa {
   font-family: 'nta';
   color:white;
-}
\ No newline at end of file
+}
+
+
+.bordered-output-present{
+    border: 1px solid #bfc1c3;
```

Comment:
These new lines appear to be indented with 4 spaces compared to the 2 used elsewhere. Would be worth configuring your editor to match the surrounding code if it can do that